cpp/data/system_updates/123.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -o errexit
+
+if [ $TUEFIND_FLAVOUR=="ixtheo" ] 
+then
+  echo "export VUFIND_LOCAL_MODULES=TueFindSearch,TueFind,IxTheo" >> /etc/profile.d/vufind.sh
```

Comment:
please use a different command like e.g. `sed` to make sure that this statement is added in the same spot where the installer would add it to.